core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -296,15 +295,19 @@ struct ALL_t {
   constexpr bool operator==(const ALL_t&) const { return true; }
 };
 
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
+namespace Impl {
+using Kokkos::ALL_t;
```

Comment:
```suggestion
using All_t KOKKOS_DEPRECATED_WITH_COMMENT("Use Kokkos::ALL_t instead!") = Kokkos::ALL_t;
```

Reply:
Yes, I agree. It's more for people to get some notification in case they were using it (even if they were not supposed to).